.github/workflows/repo_actions.yml

```
@@ -0,0 +1,20 @@
+name: Execute Management Actions
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, edited]  # triggers when PR is opened or updated
+  issue_comment:
+    types: [created]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Run local action
+        uses: BeeStation/Repo-Actions
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          tagLabels: Administration, Balance/Rebalance, Feature, Fix, Mapping, Port, Refactor, Quality Of Life, Removal, Revert, Tweak, UI
```

should this not be TGUI-Changes instead of UI? I've never seen the UI tag used
```suggestion
          tagLabels: Administration, Balance/Rebalance, Feature, Fix, Mapping, Port, Refactor, Quality Of Life, Removal, Revert, Tweak, TGUI-Changes
```

---

The review module is kinda stupid right now and does a direct match for the text in the PR rather than a dictionary lookup so complex tags wouldn't work. Also tgui change is handled by a different bot like the github tag which is why it still works